synapse/axon.py

```diff
@@ -389,6 +472,30 @@
             await asyncio.sleep(0)
         return size
 
+    async def dels(self, sha256s):
+        return [await self.del_(s) for s in sha256s]
+
+    async def del_(self, sha256):
+
+        async with self.holdHashLock(sha256):
+
+            byts = self.axonslab.pop(sha256, db=self.sizes)
+            if not byts:
+                return False
+
+            size = int.from_bytes(byts, 'big')
+            await self.axonmetrics.set('file:count', self.axonmetrics.get('file:count') - 1)
+            await self.axonmetrics.set('size:bytes', self.axonmetrics.get('size:bytes') - size)
```

Comment:
Should we add an entry into the history db for a delete?

Reply:
Unfortunately, that seqn doesn't have sufficiently extensible enveloping to safely allow for a new record format.

Reply:
Since its just a tuple of (size, sha256), and we don't seem to ever actually operate on those (just yield them out), I was thinking we could extend it.  That being said, doesn't seem too critical.